CRM/Price/Form/Preview.php

```diff
@@ -132,7 +132,9 @@ public function buildQuickForm() {
         foreach ($group['fields'] as $field) {
           $fieldId = $field['id'];
           $elementName = 'price_' . $fieldId;
-          CRM_Price_BAO_PriceField::addQuickFormElement($this, $elementName, $fieldId, FALSE, $field['is_required']);
+          if (!empty($field['options'])) {
+            CRM_Price_BAO_PriceField::addQuickFormElement($this, $elementName, $fieldId, FALSE, $field['is_required']);
+          }
```

Comment:
could be, I just found this on initial QA. Also, not sure why the merge build isn't completing.